docs/proposals/multi-version.md

```diff
@@ -0,0 +1,128 @@
+    supported minor version. 
+
+- Basic upgrading rules
+  - A LI instance could be upgraded to the currently supported patch level of its current minor version. 
+  - If a LI instance runs on a currently supported patch level or its current minor version is not supported anymore,
+    then: 
+    - it could be upgraded to the next supported minor version or 
+    - if there is no supported higher minor version it could be upgraded to the lowest supported minor version of the 
+      next major version.
+
+- An upgrade of a LaaS landscape updates the list of supported LI versions according to the following rules:
+  - A supported minor version is set on deprecated, if there is a supported version with a higher major version or
+    a supported version with the same major but a higher minor version.
+  - A minor version could only be removed if it was deprecated for more than 3 months.
+  - It is allowed to replace a minor version with one with a higher patch level but not with a lower one. Such a change 
+    will not affect if the minor version was deprecated or not.
+  - It is not allowed to add a new supported version with a lower minor level than the highest supported minor level of 
+    the same major level.
+  - It is not allowed to remove a minor version which is the next version of another supported minor version according
+    to the update rules above.
+
+- Upgrading LIs
+  - Every LI in a LaaS landscape is automatically upgraded to the currently supported patch version of its current
+    minor version.
+  - A customer could decide if his LI instance is upgraded automatically to higher minor versions or if this should be
+    triggered manually.
+  - Upgrades to the next major version must be triggered manually.
+  - Customers are automatically informed about newly supported and deprecated LI component versions. 
+  - Customers are automatically informed when a LI version will be removed from the supported version list.
+  - If a customer LI instance is running on an unsupported minor version, it is automatically upgraded,
+    according to upgrading rules described above. The customer is automatically informed about this.
+
+- A customer must be able 
+  - to select its intended LI version during onboarding. This must be optional. He gets the latest version
+    if nothing is selected.
+  - to see
+    - the current version of his LI
+    - the supported versions
+    - the deprecated supported versions
+    - the exact timeframes when deprecated LI versions are removed (-> forced update)
+    - the upgrade path according to the rules above
+
+## Questions:
+
+- If we release new minor versions too frequently, the customer must test and upgrade quite often.
+
+## Potential architectural consequences
+
+- The Central Landscaper should always be on the latest Landscaper version. The version referenced by the
+  LaaS project.
```

Comment:
This means we can't have breaking changes in the LI blueprints, right? Or the landscaper has to provide a conversion of incompatible blueprints etc.

Reply:
I wanted to express only that the Landscaper version of the central landscaper is independent of the supported versions of the LI component. I changed the wording.